### ISSUE TITLE
Medevac Helicopter displays -20 trillion pounds of maximum lift

### ISSUE THREAD

**Issue by RobbieNeko** [cataclysmbnteam/Cataclysm-BN#5258]
### Describe the bug

A medevac helicopter seen in my personal playthrough displays a casual -20 trillion pounds of maximum lift.
I sincerely doubt the helicopter has such downforce

### Steps To Reproduce

1. Load into save provided
2. Examine helicopter
3. Laugh

### Screenshots

![image](https://github.com/user-attachments/assets/3ab91cda-aa54-420a-a632-84e5aa244ccc)

### Versions and configuration

- OS: Linux
    - Kernel Version: 6.10.6; Distributor ID: Fedora; Description: Fedora Linux 40 (KDE Plasma); Release: 40;
- Game Version: 8db8a67 [64-bit]
- Graphics Version: Tiles
- LAPI Version: 2
- Game Language:  []
- Mods loaded: [
    Bright Nights [bn],
    Disable NPC Needs [no_npc_food],
    Simplified Nutrition [novitamins],
    No Rail Stations [No_Rail_Stations],
    Limit Fungal Growth [limit_fungal_growth],
    Item Category Overhaul [Item_Category_Overhaul],
    Battle Maid Redux [battle_maid],
    Arcana and Magic Items [Arcana],
    Magical Nights [MagicalNights],
    Monster Girls Mutations [MonsterGirls],
    Pride Flags for BN [pride_flags],
    Expanded Archery [ExArch],
    Arcana/Magical Nights Patchmod [Arcana_MagiclalNights_Patch],
    More Survivor Stuff [more_survivor_stuff],
    Faster Stamina Regen [stamina_regen_buff],
    Stronger archery [bow_opness],
    Stats Through Kills [stats_through_kills]
]

### Additional context

[World / Save file (zipped)](https://github.com/user-attachments/files/16769486/Ellysium.zip)

**Comment by scarf005** [comment]
_pounds????????_

**Comment by RoyalFox2140** [comment]
It needs to be turned on once to go away, but it still shouldn't be displaying a number like this if it was never ran.

**Comment by scarf005** [comment]
related: https://github.com/cataclysmbnteam/Cataclysm-BN/pull/1631#issuecomment-1160897050

this is a known but an relatively low-priority issue.

**Comment by JReese3** [comment]
Can it go into orbit...?  Like Kerbal Space Program?

**Comment by scarf005** [comment]
> Can it go into orbit...? Like Kerbal Space Program?

no. BN has fixed height limit, which is up to +10 and down to -10 z level.

**Comment by ZChris13** [comment]
rotor lift calcs are all screwed up, loading a helicopter from a save or from the map gives a different lift and max air speed when compared to installing rotors on an existing vehicle
this might be related

**Comment by scarf005** [comment]
helicopter lifts are cached, it might be related to lift cache invalidation
